include/Isd.h

```diff
@@ -59,19 +61,17 @@ namespace ale {
     double starting_ephemeris_time;
     double center_ephemeris_time;
 
-    double t0_ephemeris_time;
-    double dt_ephemeris_time;
-
-    double t0_quaternions;
-    double dt_quaternions;
-
     json naif_keywords;
+    
+    PositionInterpolation interpMethod;
```

Comment:
Should there also be a RotationInterpolation here?

Reply:
Presumably, but I don't think we currently don't have a key for it. Now that I think about it am not sure we really need to pack the interpolation method? Probably leave as is until it is part of the JSON ISD